<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <rosparam file="$(find ackermann_nav)/param/ackermann_teleop_param.yaml" command="load" />

  <node name="ackermann_teleop_original" pkg="ackermann_nav" type="ackermann_teleop_original" output="screen"/>

</launch>

